google/codelab/provision_spinnaker_codelab_image.sh

```diff
@@ -20,18 +20,18 @@
 # specifically for creating a new (running) codelab image out of a fresh
 # spinnaker image.
 
+if ! hal --version; then
+  echo "Spinnaker is not installed. Look at http://www.spinnaker.io for installation instructions."
```

Comment:
Do you mean "Spinnaker" or "halyard"?
Also s/Look at/Refer to/

Reply:
Fixed